src/ImportDefinitionsBundle/Resources/config/services/commands.yml

```diff
@@ -21,4 +21,22 @@ services:
             - '@import_definitions.repository.export_definition'
             - '@import_definition.exporter'
         tags:
-            - { name: 'console.command', command: 'export-definitions:export' }
\ No newline at end of file
+            - { name: 'console.command', command: 'export-definitions:export' }
+
+    ImportDefinitionsBundle\Command\ImportImportDefinitionCommand:
+        arguments:
+            - '@import_definitions.metadata.import'
+            - '@import_definitions.repository.definition'
+            - '@pimcore.dao.object_manager'
+            - '@coreshop.resource_controller.form_factory'
+        tags:
+            - { name: 'console.command', command: 'import-definitions:import-definition' }
+
+    ImportDefinitionsBundle\Command\ImportExportDefinitionCommand:
+        arguments:
+            - '@import_definitions.metadata.export'
```

Comment:
`"@=service('coreshop.resource_registry')->get('import_definitions.export_definition')"`

Use an expression here instead of creating a new service